packages/zoe/src/contracts/priceAggregatorChainlink.js

```diff
@@ -347,10 +351,13 @@ const start = async (
    * @param {bigint} blockTimestamp
    */
   const oracleInitializeNewRound = (_roundId, _oracle, blockTimestamp) => {
-    if (!newRound(_roundId)) return;
+    if (!newRound(_roundId)) {
+      return;
+    }
     const lastStarted = oracleStatuses.get(_oracle).lastStartedRound; // cache storage reads
-    if (_roundId <= add(lastStarted, restartDelay) && lastStarted !== 0n)
+    if (_roundId <= add(lastStarted, restartDelay) && lastStarted !== 0n) {
       return;
+    }
```

Comment:
This fixes a hazard that motivates the `curly` rule. (The other motivation is simplifying the Jessie grammar, I think.)

Reply:
yes:
> In Jessie, only blocks are accepted as arms

https://github.com/agoric-labs/jessica/blob/e8ab6f70065360e201d5230824796b1ce6557cb7/lib/quasi-jessie.js.ts#L141

Reply:
The motivation is writing reliable code. Before I religiously followed this style I would occasionally screw up by adding a statement to an indented block, not noticing that the indented block was not delimited by curlies.

There are also some strange language edge cases associated with curly-less control-flow arms. For example, what if it brings a variable into scope? What is the scope of the variable? What looks like the scope of the variable? Bad for these two answers to disagree.